notebooks/line_of_sight_tutorial.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>overall, this notebook deserves a bit more explanations of what it aims to produce</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/sim-pipeline/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>